uint16 gpioId
uint8 dutycycle
---
bool result